rr_platform/src/motor_relay_node/motor_relay_node.cpp

```diff
@@ -96,6 +121,12 @@ int main(int argc, char **argv) {
     nhp.param(std::string("steering_topic"), steering_topic_name, std::string("/steering"));
     ros::Subscriber steering_sub = nh.subscribe(steering_topic_name, 1, SteeringCallback);
 
+    nhp.param(std::string("p_const"), p_const, 0.0);
```

Comment:
While these default values are sensible for the type of the variable, they are pretty useless in practice. 
Your ROS parameters should have useful default values. Under normal conditions, average users should be able to get at least minimal performance out of your node with default parameter values. Here, that might mean poorly tuned, but still functional PID constants (like P = 1 or something similar).
If there truly exist no useful default values for a parameter, then it should be a required parameter with no default value specified.